datalad/interface/common_opts.py

```diff
@@ -269,6 +269,11 @@
     See https://git-annex.branchable.com/git-annex-groupwanted/ for more information""",
     constraints=EnsureStr() | EnsureNone())
 
+annex_ignore_opt = Parameter(
```

Comment:
Is there another envision use for this option in another command? If not, I think this should move into the `siblings` code. It is hard to get this description right if the usage patterns are unclear. And it is easy to break semantics when later fixing up the description from one angle, without seeing other potential usage. UNtil this is clear there is little value in centralization IMHO.